react-relay/index.d.ts

```diff
@@ -128,7 +146,13 @@ declare module "react-relay" {
     }
 
     interface RelayProp {
-        variables: any
-        setVariables(variables: Object, onReadyStateChange?: OnReadyStateChange): void
+        route: { name: string; }; // incomplete, also has params and queries
+        variables: any;
+        pendingVariables?: any;
+        setVariables(variables: Object, onReadyStateChange?: OnReadyStateChange): void;
+        forceFetch(variables: Object, onReadyStateChange?: OnReadyStateChange): void;
```

Comment:
We can go full 2.2 if you are already on 2.2. It involves changing the header so the tool verifies with 2.2. The tool will also publish an old version for compat with 2.1. 

In this case use lower case object